src/components/explorer/IndividualsTable.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import { withBasePath } from "../../utils/url";
+import ExplorerSearchResultsTable from "./ExplorerSearchResultsTable";
+
+const IndividualRender = ({individual}) => {
+    const alternateIds = individual.alternate_ids ?? [];
+    const listRender = alternateIds.length ? " (" + alternateIds.join(", ") + ")" : "";
+    return (
+        <>
+            <Link
+                to={(location) => ({
+                    pathname: withBasePath(`data/explorer/individuals/${individual.id}/overview`),
+                    state: { backUrl: location.pathname },
+                })}
+            >
+                {individual.id}
+            </Link>{" "}
```

Comment:
What is `{" "}` for?

Reply:
forces a space after a closing tag because of the way react handles spaces in JSX